agent/pairwise/pairwise.go

```diff
@@ -67,27 +67,13 @@ func NewCalleePairwise(msgFactor didcomm.MsgFactor, agent ssi.Agent,
 }
 
 func (p *Callee) CheckPreallocation(cnxAddr *endp.Addr) {
-	a := p.agent.(comm.Receiver)
```

Comment:
What does this mean for use of `cnxAddr.EdgeToken` for preallocation in general? (sorry, I haven't time to check that by myself...)

Reply:
Actually we shouldn't need it anymore, I think 🤔

Reply:
maybe you could try to remove it if you have time... It would be good to have obsolete lines removed. If you did it already I didn't notice it.

Reply:
Apparently the change broke something (surprise 😄 ) need to recheck.